shared/js/l10n.js

```diff
@@ -1940,17 +1940,7 @@
 
     var entity = this.ctx.getEntity(l10n.id, l10n.args);
 
-    var value;
-    if (entity.attrs && entity.attrs.innerHTML) {
-      // XXX innerHTML is treated as value (https://bugzil.la/1142526)
-      value = entity.attrs.innerHTML;
-      console.warn(
-        'L10n Deprecation Warning: using innerHTML in translations is unsafe ' +
```

Comment:
Why? It seems to me that we can start treating innerHTML the same way as we'd be treating entity.onclick. Why special case it?